src/transformers/models/mixtral/modeling_mixtral.py

```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-# Copyright 2023 Mistral AI and the HuggingFace Inc. team. All rights reserved.
+# Copyright 2023 Mixtral AI and the HuggingFace Inc. team. All rights reserved.
```

Comment:
The name of the company is Mistral AI and not Mixtral AI.
Also see: https://mistral.ai/company/

This line should not be changed IMO.

Reply:
Some of the documentation was CTRL+H replaced by accident when I was making my changes and should be restored, yeah